### PR TITLE
introduce %timestamp%

### LINKED ISSUE #249: Introduce keyword %timestamp%
For proper support of spells in Aqua AIR needs an support of timestamp that could be obtained from a particle. Timestamp is a `u64` value that would be pass in a "standard manner" for the interpreter via `RunParameters`.

### PR DESCRIPTION
**Pull request by mikevoronov**
Resolves #249.

### CODE REVIEW

File: crates/air-lib/interpreter-interface/src/run_parameters.rs [hunk 1]
```diff
@@ -27,4 +29,28 @@ pub struct RunParameters {
...
+        }
+    }
+
+    pub fn into_ivalue(self) -> IValue {
+        let run_parameters = vec![
+            IValue::String(self.init_peer_id),
+            IValue::String(self.current_peer_id),
+            IValue::U64(self.timestamp),
+        ];
+        let run_parameters = NEVec::new(run_parameters).unwrap();
```
Comment (ValeryAntopol) [inline crates/air-lib/interpreter-interface/src/run_parameters.rs:53]: do we need a comment about this unwrap?

File: avm/server/src/runner.rs [hunk 2]
```diff
@@ -133,7 +131,7 @@ fn prepare_args(
         IValue::String(air.into()),
         IValue::ByteArray(prev_data.into()),
         IValue::ByteArray(data.into()),
-        IValue::Record(run_parameters),
+        run_parameters,
```
Comment (ValeryAntopol) [inline avm/server/src/runner.rs:134]: From the changes i see in this PR, I do not understand why this change is needed or possible

**Review by ValeryAntopol — APPROVED** [review]

File: air/src/execution_step/execution_context/context.rs [hunk 3]
```diff
@@ -96,9 +87,29 @@ impl<'i> ExecutionCtx<'i> {
...
 }
 
+use serde::Deserialize;
+use serde::Serialize;
 use std::fmt::Display;
 use std::fmt::Formatter;
 
+/// It reflects RunParameters structure due to limitation of the marine macro to support Rc.
+#[derive(Debug, Default, Clone, Eq, PartialEq, Serialize, Deserialize)]
+pub(crate) struct RcRunParameters {
```
[the remark this answers is not in the document]
Comment (mikevoronov) [reply air/src/execution_step/execution_context/context.rs:97]: Hmm, don't know tbh, I think that `Rc` prefix makes more sense then `Light` suffix, because one could interpret `Light` differently. Let's stick with `Rc` until we come out with a better idea.

File: avm/server/src/runner.rs (hunk 2 above)
Comment (mikevoronov) [reply avm/server/src/runner.rs:134]: Discussed in private.

File: avm/client/src/importObject.ts [hunk 4]
```diff
@@ -0,0 +1,4 @@
+// auto-generated
+
+export const __wbg_callserviceimpl = "";
+export const __wbg_getcurrentpeeridimpl = "";
```
[the remark this answers is not in the document]
Comment (mikevoronov) [reply avm/client/src/importObject.ts:4]: It was changed by accident, thanks.
